Source/Session.swift

```diff
@@ -252,10 +261,12 @@ open class SessionManager {
 
     open func upload(multipartFormData: @escaping (MultipartFormData) -> Void,
                 usingThreshold encodingMemoryThreshold: UInt64 = MultipartUpload.encodingMemoryThreshold,
+                fileManager: FileManager = .default,
                 with request: URLRequestConvertible) -> UploadRequest {
         let multipartUpload = MultipartUpload(isInBackgroundSession: (session.configuration.identifier != nil),
                                               encodingMemoryThreshold: encodingMemoryThreshold,
                                               request: request,
+                                              fileManager: fileManager,
                                               multipartBuilder: multipartFormData)
```

Comment:
Cool. No reason to not allow users to use the top-level APIs with custom file managers if needed. Especially since you're leveraging default values. I dig it.